src/Actions/babylon.actionManager.ts

```diff
@@ -138,9 +143,8 @@
             return ActionManager._OnKeyUpTrigger;
         }
 
-        public static DragMovementThreshold = 10; // in pixels

```

Comment:
I removed them because I needed the same kind of values for Observable. 
1. I could have used ActionManager values but I thought it was not correct ; why does Observable need to rely on ActionManager values ? 
2. I also could have create the same value in Observable class but I thought it was redundant. Is it really needed to provide 2 delays for double click, one for ActionManager and the other one for Observable ?
3. So I chose to deport theses values in Scene class because I considered that it was a good place to share theses datas within both classes.
What kind of solution do you prefer ?